packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -1149,13 +1151,15 @@ class Auth {
      */
     async getPasswordLessAccessToken(parameters: LoginPasswordlessParams) {
         const pwdlessLoginToken = parameters.pwdlessLoginToken
+        const dntPref = this.getDnt({includeDefaults: true})
         const token = await helpers.getPasswordLessAccessToken(
             this.client,
             {
                 clientSecret: this.clientSecret
             },
             {
-                pwdlessLoginToken
+                pwdlessLoginToken,
+                dnt: dntPref !== undefined ? String(dntPref) : undefined
```

Comment:
out of curiosity, is there a reason why dnt is a string here and a boolean for `loginIDPUser` ?

Reply:
It's because commerce-sdk-isomorphic expects a string there for some reason, and it's not being consistent: https://github.com/SalesforceCommerceCloud/pwa-kit/pull/2248#discussion_r1951532532

Reply:
@hajinsuha1 Yeah for the `getPasswordLessAccessToken` helper the param expects a string whereas the other ones expect a boolean.